packages/binding-coap/src/coap.ts

```diff
@@ -29,16 +27,25 @@ export * from "./coap-client";
 export * from "./coaps-client-factory";
 export * from "./coaps-client";
 
-export class CoapOption {
-    public "cov:optionName": OptionName;
-    public "cov:optionValue": OptionValue;
-}
-
 export type CoapMethodName = "GET" | "POST" | "PUT" | "DELETE" | "FETCH" | "PATCH" | "iPATCH";
 
+export type BlockSize = 16 | 32 | 64 | 128 | 256 | 512 | 1024;
+
+export type BlockSizeOptionValue = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface BlockWiseTransferParameters {
+    "cov:block2SZX"?: BlockSize;
+    "cov:block1SZX"?: BlockSize;
+}
+
 export class CoapForm extends Form {
-    public "cov:methodName"?: CoapMethodName;
-    public "cov:options"?: Array<CoapOption> | CoapOption;
+    public "cov:method"?: CoapMethodName;
```

Comment:
Regarding `cov:method`: As the HTTP binding is using `htv:methodName` (see [here](https://w3c.github.io/wot-binding-templates/bindings/protocols/http/#http-vocabulary-terms)), I was wondering if we should also keep `cov:methodName` for the sake of consistency.

Reply:
Consistency is important!
I hope the binding documents do align there as well... which does not seem to be the case.
* https://w3c.github.io/wot-binding-templates/bindings/protocols/http/index.html#http-vocabulary-terms
* https://w3c.github.io/wot-binding-templates/bindings/protocols/coap/index.html#requests-responses

@egekorkan

Reply:
The naming of `cov:method` is consistent with [`modbus:function`](https://w3c.github.io/wot-binding-templates/bindings/protocols/modbus/#form-terms) and [`mqv:controlPacket`](https://w3c.github.io/wot-binding-templates/bindings/protocols/mqtt/#controlpacket), so I don't see any need for change here.